app/routes.py

```diff
@@ -1 +1,244 @@
-from flask import Blueprint
\ No newline at end of file
+from flask import Blueprint, jsonify, abort, make_response, request
+from app import db
+from app.models.task import Task
+from sqlalchemy import asc, desc
+from datetime import date, time, datetime
+import json
+import requests
+import os
+from dotenv import load_dotenv
+from app.models.goal import Goal
+
+load_dotenv()
+
+task_bp = Blueprint("tasks", __name__, url_prefix="/tasks")
+goal_bp = Blueprint("goals", __name__, url_prefix="/goals")
+
+## Validate Model ##
+def validate_model(cls, model_id):
+    try:
+        model_id = int(model_id)
+    except:
+        abort(make_response({"message": f"{model_id} is not a valid type ({type(model_id)}). Must be an integer)"}, 400))
+
+    model = cls.query.get(model_id)
+    
+    if not model:
+        abort(make_response({"message": f"{cls.__name__} {model_id} does not exist"}, 404))
+        
+    return model
+        
+## CREATE
+@task_bp.route("", methods=['POST'])
+def create_task():
+    request_body = request.get_json()
+    
+    if not request_body.get('title') or not request_body.get('description'):
+        abort(make_response({"details": "Invalid data"}, 400))
+        
+    task = Task.from_dict(request_body)
+    
+    db.session.add(task)
```

Comment:
Great job using `db.session.<method-name>`. Session gives us access to the follow:

- `db` is our app’s instance of SQLAlchemy.
- `session` represents our active database connection.
- By referencing `db.session` we can use SQLAlchemy’s methods to perform tasks like:
    - committing a change to a model
    - storing a new record of a model
    - deleting a record.
- By referencing `db.session.add()` you are able to use the SQLAlchemy method to store a new record of the task model